spec/request/api/v1/targets/compatibles_target_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+describe 'GET /api/v1/targets/compatibles', type: :request do
```

Comment:
Strictly... this test should be just testing the endpoint, that means only testing the logic on the endpoint and what is expected to be returned in the views. Like a normal index endpoint in this case. 

Anyway, it's ok, dont change it! But checking that the logic about matching the compatibles targets works should be tested in a model test

Reply:
Also I see that in your tests, you just use the topic to know if a target is compatible or not, I would a target in a really far distance to make sure that the filter by distance is working too